### PR TITLE
AppVeyor: Minor changes

### PR DESCRIPTION
**Pull request by Margen67**
Add yml validater to top.
Skip files unrelated to build.
Reformat comments.
Only build Release since Azure Pipelines already builds Debug and is faster.
Remove cmd since it's the default shell, and unneeded characters.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
-  # If this is modified, please also update the build script
+image: # If this is modified, please also update the build script
   - Visual Studio 2019
 
-configuration:
-  # The builds will be run in this order
+configuration: # The builds will be run in this order
   - Release
-  - Debug
+#  - Debug
```
Comment (GXTX) [inline .appveyor.yml]: Even if you say it's faster on Azure, I'm not sure I see a reason to not build here? We suggest users to download artifacts *from* AppVeyor.

**Comment by GXTX** [comment]
All of these tiny changes to each different CI, we should be consolidating to 1 as it just adds additional complexity and requires maintainers to have org accounts open for each one of these different services.

File: .appveyor.yml (hunk 1 above)
Comment (Margen67) [reply .appveyor.yml]: Debug doesn't have artifacts so having it on AppVeyor is redundant.

**Comment by Margen67** [comment]
@GXTX I'm not sure what you mean: 
image

**Comment by GXTX** [comment]
CXBX-R currently has 3 different CI providers (AppVeyor, Travis-Ci, Azure) which adds support complexity. Even just recently you've been going back to modify and change small issues with each provider. It would seem worth it to cut support for at least 2 of those and instead move to a provider which is giving up-to-date images and software.

File: .appveyor.yml (hunk 1 above)
Comment (GXTX) [reply .appveyor.yml]: Missed that, thanks.

**Comment by Margen67** [comment]
> 
> 
> CXBX-R currently has 3 different CI providers (AppVeyor, Travis-Ci, Azure) which adds support complexity. Even just recently you've been going back to modify and change small issues with each provider. It would seem worth it to cut support for at least 2 of those and instead move to a provider which is giving up-to-date images and software.

I agree. Travis should be first since it doesn't have artifacts, and only supports VS2017.

**Comment by GXTX** [comment]
> (or better yet, GitHub Actions) provide a way to link to the latest artifacts.

There is a way to direct link however users needs to be logged into GitHub to access them (See https://github.com/actions/upload-artifact/issues/51), this can be mitigated by using releases instead - although somewhat dirty it would be possible to "keep it clean" by using *more actions* to delete non-tagged releases.

**Comment by Margen67** [comment]
Not much point to this if it might be removed soon: #1848

**Comment by Margen67** [comment]
Reopening since AppVeyor keeps getting triggered by my PR: #1854

**Pull request merged**